docs/tx.md

```diff
@@ -36,6 +36,12 @@ content locally and on Transifex.
   resource you wish to pull down, usually resources where translation is complete.
   Commit result, create PR, and merge PR after Travis build has passed.
 
+- `pullAll`- should be run while on a fresh branch, separate from other
+  code changes. It will grab translated content from all resources for the project it is run in in last updated resource order. As each resource it is pulled it merges pulled content into `src/trns/po/{localeCode}.po` files. If you'd like to have a history of each change
+  based on resource add a flag of `-c` and it will create a unique commit for each resource
+  that has a change to commit.
+  Commit result, create PR, and merge PR after Travis build has passed.
```

Comment:
I would _love_ to automate this. We're actually already using a Node package that interacts with the GitHub API for [the `gh status` command](https://github.com/meetup/mwp-cli/blob/master/src/commands/ghCommands/status.js) - I don't actually think it would be too difficult to add a `gh pr create` command so that the developer is only responsible for merging the PR once the build succeeds...

I suspect you have enough on your plate, but that is where I'm planning to take this feature next

Reply:
We have a ticket actually to make a jenkins job (On @burrata once this is merged) so that Camila can at least trigger and build and generate a PR. I think once we are comfortable with the output after a few runs we can just merge to master. https://meetup.atlassian.net/browse/MW-3109

Reply:
ah, excellent - i'll add notes to that ticket so we don't duplicate work :)